test/missing.jl

```diff
@@ -441,6 +441,17 @@ end
     @test collect(x) == [1, 2, 4]
     @test collect(x) isa Vector{Int}
 
+    @testset "length" begin
+        allmiss = Vector{Union{Int,Missing}}(missing, 10)
+        @test (length∘skipmissing)(allmiss) == 0
+
+        somemiss = [1, missing, 2, 3, 4, missing, 5, 6, 7, 8, 9, 10]
+        @test (length∘skipmissing)(somemiss) == 10
+
+        nomiss = rand(1:10, 10)
+        @test (length∘skipmissing)(nomiss) == 10
```

Comment:
one small thing - maybe add a test for the case that fails because `skipmissing` wraps some iterator that is not an array?

Reply:
Addressed in 5db9504f5c32ac207b8cdb1ce725dc7e46c1920f. Does this look good?

Reply:
OK